src/modules/search.ts

```diff
@@ -16,32 +16,32 @@ export interface SearchQuoteYahoo {
   newListingDate?: Date; // "2021-02-16"
 }
 export interface SearchQuoteYahooEquity extends SearchQuoteYahoo {
-  quoteType: "EQUITY";
-  typeDisp: "Equity";
+  quoteType: "EQUITY" | "FUTURE";
+  typeDisp: "Equity" | "Future";
 }
 export interface SearchQuoteYahooOption extends SearchQuoteYahoo {
-  quoteType: "OPTION";
-  typeDisp: "Option";
+  quoteType: "OPTION" | "FUTURE";
+  typeDisp: "Option" | "Future";
 }
 export interface SearchQuoteYahooETF extends SearchQuoteYahoo {
-  quoteType: "ETF";
-  typeDisp: "ETF"; // "Option"
+  quoteType: "ETF" | "FUTURE";
+  typeDisp: "ETF" | "Future"; // "Option"
 }
 export interface SearchQuoteYahooFund extends SearchQuoteYahoo {
-  quoteType: "MUTUALFUND";
-  typeDisp: "Fund";
+  quoteType: "MUTUALFUND" | "FUTURE";
+  typeDisp: "Fund" | "Future";
 }
 export interface SearchQuoteYahooIndex extends SearchQuoteYahoo {
-  quoteType: "INDEX";
-  typeDisp: "Index";
+  quoteType: "INDEX" | "FUTURE";
+  typeDisp: "Index" | "Future";
 }
 export interface SearchQuoteYahooCurrency extends SearchQuoteYahoo {
-  quoteType: "CURRENCY";
-  typeDisp: "Currency";
+  quoteType: "CURRENCY" | "FUTURE";
+  typeDisp: "Currency" | "Future";
 }
 export interface SearchQuoteYahooCryptocurrency extends SearchQuoteYahoo {
-  quoteType: "CRYPTOCURRENCY";
-  typeDisp: "Cryptocurrency";
+  quoteType: "CRYPTOCURRENCY" | "FUTURE";
+  typeDisp: "Cryptocurrency" | "Future";
```

Comment:
rather than getting all the other types to accept "future" as well, you should rather make a new interface specifically for that type below them, e.g. `SearchQuoteYahooFuture`, and then add that type to the end of the list of quotes types accepted by SearchResult.quotes (should be line 74).

Reply:
Sure! Makes sense.